src/main/java/captain/cybot/adventure/backend/component/DBInit.java

```diff
@@ -74,12 +78,92 @@ private void initDefaultCosmetics() {
         }
     }
 
-    private void initQuestions() {
-        // EXAMPLE for the future reference
-//        String[][] crosswordBlock = {{"1", "X", "X"}, {"|", "|", "|"}, {"2", "X", "|"}};
-//        String[] hints = {"Denotes one or more people already mentioned and assumed to be common " +
-//                "knowledge", "perform an action"};
-//        String[] answer = {"the", "do"};
-//        questionRepository.save(new Crossword(crosswordBlock, hints, answer));
+    private void initDefaultQuestions() {
+        QuestionOrder w1Q1 = questionOrderRepository.findByPlanetAndQuestionNumber("EARTH", 1);
+        QuestionOrder w1Q2 = questionOrderRepository.findByPlanetAndQuestionNumber("EARTH", 2);
+        QuestionOrder w1Q3 = questionOrderRepository.findByPlanetAndQuestionNumber("EARTH", 3);
+        QuestionOrder w1Q4 = questionOrderRepository.findByPlanetAndQuestionNumber("EARTH", 4);
+        QuestionOrder w2Q1 = questionOrderRepository.findByPlanetAndQuestionNumber("MARS", 1);
+        QuestionOrder w2Q2 = questionOrderRepository.findByPlanetAndQuestionNumber("MARS", 2);
+        QuestionOrder w2Q3 = questionOrderRepository.findByPlanetAndQuestionNumber("MARS", 3);
+        QuestionOrder w2Q4 = questionOrderRepository.findByPlanetAndQuestionNumber("MARS", 4);
+        QuestionOrder w3Q1 = questionOrderRepository.findByPlanetAndQuestionNumber("NEPTUNE", 1);
+        QuestionOrder w3Q2 = questionOrderRepository.findByPlanetAndQuestionNumber("NEPTUNE", 2);
+        QuestionOrder w3Q3 = questionOrderRepository.findByPlanetAndQuestionNumber("NEPTUNE", 3);
+        QuestionOrder w3Q4 = questionOrderRepository.findByPlanetAndQuestionNumber("NEPTUNE", 4);
+        QuestionOrder w4Q1 = questionOrderRepository.findByPlanetAndQuestionNumber("JUPITER", 1);
+        QuestionOrder w4Q2 = questionOrderRepository.findByPlanetAndQuestionNumber("JUPITER", 2);
+        QuestionOrder w4Q3 = questionOrderRepository.findByPlanetAndQuestionNumber("JUPITER", 3);
+        QuestionOrder w4Q4 = questionOrderRepository.findByPlanetAndQuestionNumber("JUPITER", 4);
```

Comment:
TODO: change order of of questions to be:
1) word scramble
2) guess the image
3) word search
4) crosssword